notification/twilio/voice.go

```diff
@@ -54,6 +54,7 @@ const (
 	digitConfirm  = "3"
 	digitOldAck   = "8"
 	digitOldClose = "9"
+	digitEsc      = "5"
```

Comment:
Same, let's spell out escalate so it's not confused as an "escape digit" or something like that
```suggestion
	digitEscalate      = "5"
```